### PR TITLE
Failover database and elastic pool swagger changes

### PR DESCRIPTION
**Pull request by emgu-ms**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [x] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [x] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

Description:
Changes to add failover database and failover elastic pool to swagger.

### CODE REVIEW

**Review by jaredmoo — APPROVED** [review]
When the api is available in production, then you need to add links to these files in readme.md

File: specification/sql/resource-manager/Microsoft.Sql/preview/2018-06-01-preview/FailoverDatabases.json [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{serverName}/databases/{databaseName}/failover": {
+      "post": {
+        "tags": [
+          "FailoverDatabases"
+        ],
+        "description": "Failovers a database.",
```
[the remark this answers is not in the document]
Comment (emgu-ms) [reply specification/sql/resource-manager/Microsoft.Sql/preview/2018-06-01-preview/FailoverDatabases.json]: API is available in production, I will go ahead and add to the readme.md.  For failover database, customers know what a failover is.  We originally had restart database but opted for failover instead as in the future we're thinking about doing planned and unplanned failovers

[the remark this answers is not in the document]
Comment (jaredmoo) [reply specification/sql/resource-manager/Microsoft.Sql/preview/2018-06-01-preview/FailoverDatabases.json]: Ok in that case please add to all package-composite-v*, and also the package-pure- which matches the api version.

**Comment by nschonni** [comment]
Avocado failures are because the files have a BOM. You'd need to resave as regular UTF-8-without BOM to fix it

**Comment by sergey-shandar** [comment]
@daschult could you check JS SDK generation failure?

**Comment by daschult** [comment]
@sergey-shandar We've disabled the JavaScript portion of Swagger to SDK in favor of the JavaScript portion of SDK Automation (which is green in the [SDK Automation comment](https://github.com/Azure/azure-rest-api-specs/pull/6365#issuecomment-502864846) above). I recommend disabling the JavaScript Swagger to SDK check for this repository.

**Comment by ravbhatnagar** [comment]
Signing off from ARM side.

**Comment by yaohaizh** [comment]
@emgu-ms could help resolve the conflict first?

**Comment by yaohaizh** [comment]
@emgu-ms Any update on this? Could you help resolve the conflict?

**Comment by emgu-ms** [comment]
> @emgu-ms Any update on this? Could you help resolve the conflict?
On it, sorry was oof until today

**Comment by emgu-ms** [comment]
@yaohaizh Merge conflicts resolved, thank you!

**Review by yaohaizh — APPROVED** [review]

**Pull request merged**